### PR TITLE
adding pkg_hardware_config arg

### PR DESCRIPTION
**Pull request by bbrito**
add pkg_hardware_config arg to remove the dependency to cob_harware_config as done for other arguments

### CODE REVIEW

File: cob_bringup/drivers/ur.launch [hunk 1]
```diff
@@ -33,6 +34,7 @@
 	<include if="$(arg sim)" file="$(find cob_bringup)/controllers/generic_controller.launch">
 		<arg name="robot" value="$(arg robot)"/>
 		<arg name="component_name" value="$(arg component_name)"/>
+		<arg name="pkg_hardware_config" value="$(arg pkg_hardware_config)"/>
```
Comment (fmessmer) [inline cob_bringup/drivers/ur.launch:37]: what do you need this for?
why can't you adjust the yaml's in `cob_hardware_config`?

Comment (bbrito) [reply cob_bringup/drivers/ur.launch:37]: If I want to launch a configuration from a specific project. That is why all the other files in the chain have it, isn't it?

**Review by ipa-rmb — APPROVED** [review]

**Comment by ipa-rmb** [comment]
Ok with you @ipa-fxm to merge?

**Comment by ipa-rmb** [comment]
Ok with you @ipa-fxm to merge?

**Review by fmessmer — COMMENTED** [review]
I don't care

**Pull request merged**